src/libcrun/container.c

```diff
@@ -3706,6 +3707,131 @@ libcrun_container_update_from_values (libcrun_context_t *context, const char *id
   return ret;
 }
 
+void populate_array_field(const char*** field, const char* array[], size_t numElements){
+    *field = malloc(numElements * sizeof(char *));
+    size_t i;
+    for (i = 0; i < numElements; i++) {
+        (*field)[i] = strdup(array[i]);
+    }
+
+    (*field)[i] = NULL;  // Terminate the array with NULL
+}
+
+int libcrun_container_print_features(struct features_info_s *info)
```

Comment:
this function is not really printing anything, let's rename to `libcrun_container_get_features`